src/main/java/de/uni_potsdam/hpi/asg/common/breeze/model/xml/Components.java

```diff
@@ -51,8 +51,8 @@ public static Components readIn(String filename) {
 			JAXBContext jaxbContext = JAXBContext.newInstance(Components.class);
 			Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
 			if(filename == null || filename.equals("")) {
-				URL url = Components.class.getResource(injarfilename);
```

Comment:
This answer describes it, although I'm not sure if this only applies if one wants to create a File from the URL/URI:
http://stackoverflow.com/a/20389418/2148890